service_info/settings/base.py

```diff
@@ -456,7 +456,7 @@
 TEXT_SAVE_IMAGE_FUNCTION = 'cmsplugin_filer_image.integrations.ckeditor.create_image_plugin'
 
 CMS_APP_NAME = 'cms'
-CMS_TOP_REL = r'cms/'
+CMS_TOP_REL = r''
```

Comment:
Do you think it's valuable to keep this setting around? Versus removing it and simplifying the urls.py use of it.